<script>
	const myPromise = new Promise((_, reject) => reject("error"))

	myPromise
		.then(result => console.log({ result })) // resolve
		.catch(error => console.log({ error })) // reject
</script>
